lib/l10n/intl_en.arb

```diff
@@ -1,11 +1,13 @@
 {
   "@@locale": "en",
   "app_emptyState_retry": "Try again",
-  "app_errorScreen_authError": "Seems like this device's authentication expired.\nMaybe signing out and in again helps?",
-  "app_errorScreen_noConnection": "We can't connect to the server.\nAre you sure you're connected to the internet?",
-  "app_errorScreen_stackTrace": "Stack trace",
-  "app_errorScreen_unknown": "Oh no! An internal error occurred:\n{error}",
-  "app_errorScreen_unknown_showStackTrace": "Show stack trace",
+  "app_error_conflict": "Your changes conflict with changes someone else did at the same time.",
```

Comment:
Maybe we can customize a meme like [this](https://imgur.com/txXrz6I) and put it here in the future?

Reply:
https://github.com/schul-cloud/schulcloud-flutter/issues/228